Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.cxx

```diff
@@ -796,7 +796,7 @@ void vtkMRMLScalarVolumeDisplayNode::CalculateAutoLevels()
 
     window = max-min;
     level = 0.5*(max+min);
-    lower = this->GetLevel();
+    lower = range[0];
```

Comment:
The behavior is different, but I assume is correct: we are now setting the initial value for the lower threshold to the minimum scalar value.

The previous behavior was setting the lower threshold to the PREVIOUS level value:

* initial:
  - window=256
  - level=128
  - threshold_min = VTK_SHORT_MIN
  - treshold_max = VTK_SHORT_MAX 
* after computeAutoWindowLevel (x1):
  - window = max-min
  - level = 0.5x(max+min)
  - threshold_min = **getlevel = 128 (previous value from initialisation)**
  - threshold_max = max
* after computeAutoWindowLevel (x2):
  - window = max-min
  - level = 0.5*(max+min)
  - threshold_min = **getlevel = 0.5x(old_max+old_min)  (previous value from 1st computeWL)**
  - threshold_max = max